heron/schedulers/src/java/com/twitter/heron/scheduler/kubernetes/KubernetesScheduler.java

```diff
@@ -358,11 +418,26 @@ protected ArrayNode getPorts(ObjectMapper mapper) {
     return ports;
   }
 
+  /**
+   * getFetchCommand
+   *
+   * Get the command that will be used to retrieve the topology JAR
+   *
+   * @return String
```

Comment:
here and elsewhere let's not specify `@return [type]` unless we have something specific to say about what's returned. The return type will be included in the javadocs without specifying `@return String`.